index.html

```diff
@@ -16,13 +16,187 @@
 </head>
 
 <body>
+  <div class="jumbotron mb-0">
+    <div class="container">
+      <div class="row">
+        <div class="col-12">
+          <h1 class="mt-3">SpaceWalkers</h1>
+        </div>
+        <div class="col-12">
+          <nav class="navbar justify-content-center">
+            <a class="mr-5" href="#">Home</a>
+            <a class="mr-5" href="#">About Us</a>
+            <a class="mr-5" href="#">Blog</a>
+            <a class="mr-5" href="#">Contact</a>
+            <div class="dropdown">
+                <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <a class="mr-5" href="#">Sign In</a>
+                </button>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                  <a class="dropdown-item" href="#">Sign Out</a>
+                </div>
+              </div>
+          </nav>
+        </div>
+      </div>
+    </div><!-- container -->
+</div>
   <div class="container">
+    <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+      <div class="carousel-inner">
+        <div class="carousel-item active">
+          <img class="d-block w-100" src="./img/slider-1.jpg" alt="First slide">
+        </div>
+        <div class="carousel-item">
+          <img class="d-block w-100" src="./img/slider-2.jpg" alt="Second slide">
+        </div>
+        <div class="carousel-item">
+          <img class="d-block w-100" src="./img/slider-3.jpg" alt="Third slide">
+        </div>
+        <div class="carousel-item">
+            <img class="d-block w-100" src="./img/slider-4.jpg" alt="Third slide">
+          </div>
+      </div>
+      <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="sr-only">Previous</span>
+      </a>
+      <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="sr-only">Next</span>
+      </a>
+    </div>
     <div class="row">
-      <div class="col-sm-12">
-        Good Luck!
+      <div class="col-12 mt-5">
+        <h2>Lift Off</h2>
+        <div class="col-12">
+          <p>As we got further and further away, it [the Earth] diminished in size. Finally it shrank to the size of a marble, the most beautiful you can imagine. That beautiful, warm, living object looked so fragile, so delicate, that if you touched it with a finger it would fall apart. Seeing this has to change a man.</p>
```

Comment:
I actually don't know the exact reason why your paragraph is off-aligned but I did find a solution. I added a `padding: 0` and it aligned everything properly!